app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -0,0 +1,2 @@
+@import 'bootstrap/scss/bootstrap';
+@import 'bootstrap-icons/font/bootstrap-icons';
```

Comment:
- [ ] If you are naming your file "application.bootstrap.scss" and you want it to serve as the main application stylesheet, you might want to rename it to "application.scss" or "application.css" since Rails, by convention, looks for these files as the main entry points for styles. This has caused an error while running your application.

### Error view:
![image](https://github.com/wickathou/Budget/assets/50990925/99190336-129e-480f-8b09-3949358dd546)


- [ ] _**To the next reviewer, kindly crosscheck the design flow as this error made sure I could not view the entire UI of the project. Thank you**_

Reply:
All fixed now, the issue was that the command `rails assets:precompile` was missing from the installation guidelines on the Readme file